mibidata/tests/test_mibi_image.py

```diff
@@ -203,10 +200,10 @@ def test_metadata_backwards_compatibility(self):
                                                       mi._DATETIME_FORMAT)
         metadata['fov_name'] = metadata['point_name']
         metadata['fov_id'] = metadata['folder'].split('/')[0]
-        del metadata['point_name']
         metadata['description'] = None
         self.assertEqual(image.metadata(), metadata)
-        self.assertEqual(len(image._user_defined_attributes), 0)
+        # point_name is retained as user-defined
+        self.assertEqual(image._user_defined_attributes, ['point_name'])
```

Comment:
I think we should try to get rid of the `point_name`, otherwise it can be confusing to have twice the same information in the MibiImage instance.
I think we can easily get rid of `point_name` in the setter. I will test a possible solution for this.

Reply:
I think we should leave it in since we are allowing arbitrary user-defined attributes it would be unexpected to exclude this particular string as one of those. Also users may have code that relies on this from the version; this gives them a warning to update without breaking anything immediately.

Reply:
I just uploaded the fix for this, shall I undo it?
Sorry, I just read the message.